pip/commands/install.py

```diff
@@ -82,6 +83,28 @@ def __init__(self, *args, **kw):
                   "regardless of what's already installed."),
         )
 
+        cmd_opts.add_option(
+            '--platform',
+            dest='platform',
+            metavar='platform',
+            default='',
+            help=("Specifically download wheels compatible with <platform> "
+                  "where the default is the platfrom of the local computer. "
+                  "This option may only be used if --download is also being "
+                  "used."),
+        )
+
+        cmd_opts.add_option(
+            '--supported-interpreter-version',
```

Comment:
`--interpreter-version` seems enough (and we are not using `--supported-platform`)